common4j/src/main/com/microsoft/identity/common/java/cache/MsalCppOAuth2TokenCache.java

```diff
@@ -148,7 +148,7 @@ public synchronized void saveCredentials(@NonNull final Credential... credential
             }
         }
 
-        saveCredentialsInternal(credentials);
+        saveCredentialsInternal(true, credentials);
```

Comment:
>saveCredentialsInternal(true, credentials); [](http://example.com/codeflow?start=8&length=43)

Can we add a test case to common that describes the difference in behavior we expected from true vs. false here?